netbox_librenms_plugin/views/__init__.py

```
@@ -10,6 +10,15 @@
 from .base.interfaces_view import BaseInterfaceTableView  # noqa: F401
 from .base.ip_addresses_view import BaseIPAddressTableView, SingleIPAddressVerifyView  # noqa: F401
 from .base.librenms_sync_view import BaseLibreNMSSyncView  # noqa: F401
+from .sync.modules import (
```

Flake8 (7.3.0)

[error] '.sync.modules.InstallBranchView' imported but unused

(F401)